### ISSUE TITLE
Test failure if we use torch 2.5

### ISSUE THREAD

**Issue by liu-jc** [SalesforceAIResearch/uni2ts#133]
**Is your feature request related to a problem? Please describe.**
If we don't require torch version < 2.5, the build test will fail, particularly test/module/test_attention.py::test_all_masked: https://github.com/SalesforceAIResearch/uni2ts/blob/2ba614de8878d350c62835c942b450d2f4d5a711/test/module/test_attention.py#L85
**Describe the solution you'd like**
The reason might be that torch 2.5 updates the function `nn.functional.scaled_dot_product_attention` (see here https://github.com/pytorch/pytorch/releases/tag/v2.5.0) 
It mentions: 
> Before this PR, fully masked rows in the attn_mask passed to nn.functional.scaled_dot_product_attention would yield NANs in the output, after this PR, fully masked rows yield 0s.

I think later maybe we would like to change the test function to assert all 0s. 
But I am not sure if we have special usage for torch.nan in output (like if we change to 0s for fully masked rows, it will break the pipeline)
@chenghaoliu89 @cuthalionn Any comments/suggestions?
